geode-connectors/src/main/java/org/apache/geode/connectors/jdbc/internal/cli/JdbcCliFunction.java

```diff
@@ -54,6 +59,11 @@ public void execute(FunctionContext<T1> context) {
     }
   }
 
+  @Override
+  public Collection<ResourcePermission> getRequiredPermissions(String regionName) {
```

Comment:
I don't think we need to add this here for an abstract function

Reply:
This is the new default that is set for any JDBC connector function as most of JDBC functions have to deal with cluster-config.
Otherwise the default would be `DATA:WRITE` inherited from Function interface.